gen/safeq

```diff
@@ -89,34 +90,34 @@ foreach my $id (sort keys %costcentersById) {
 foreach my $login (sort keys %$usersByLogin) {
 	my $attributes = $usersByLogin->{$login};
 
-	print FILE "dn: username=", $login,", ou=users, o=ysoftsafeq", "\n";
+	print FILE "dn: ", escapeDnValue("username=" . $login . ", ou=users, o=ysoftsafeq"), "\n";
 	print FILE "objectclass: ysquser", "\n";
-	print FILE "username: ", $login, "\n";
-	print FILE "ldapauthdn: cn=", $login, ", ou=users, ou=mu, dc=ucn, dc=muni, dc=cz", "\n";
-	print FILE "firstname: ", checkAttributeValue($attributes->{$A_USER_FIRST_NAME} || '(N/A)'), "\n";
-	print FILE "lastname: ", checkAttributeValue($attributes->{$A_USER_LAST_NAME} || '(N/A)'), "\n";
-	print FILE "costcenter: costcenterno=", $attributes->{$A_USER_WORKPLACE_ID} || 100, ", ou=costcenters, o=ysoftsafeq", "\n";
-	print FILE "email: ", checkAttributeValue($attributes->{$A_USER_MAIL}), "\n";
+	print FILE "username:", checkBase64($login), "\n";
+	print FILE "ldapauthdn:", checkBase64("cn=" . $login . ", ou=users, ou=mu, dc=ucn, dc=muni, dc=cz"), "\n";
+	print FILE "firstname:", checkBase64($attributes->{$A_USER_FIRST_NAME} || '(N/A)'), "\n";
+	print FILE "lastname:", checkBase64($attributes->{$A_USER_LAST_NAME} || '(N/A)'), "\n";
+	print FILE "costcenter:", checkBase64("costcenterno=" . $attributes->{$A_USER_WORKPLACE_ID} || 100 . ", ou=costcenters, o=ysoftsafeq"), "\n";
+	print FILE "email: ", checkBase64($attributes->{$A_USER_MAIL}), "\n";
```

Comment:
There is additional space after colon. In case where email needs to be encoded to base64 you will following line:
```
email: :BASE64TEXT
```